pkg/rakkess/util/printer.go

```diff
@@ -37,7 +37,18 @@ const (
 
 var IsTerminal = isTerminal
 
-func PrintResults(out io.Writer, requestedVerbs []string, results []client.Result) {
+type DisplayMode int
+
+const (
+	Icons DisplayMode = iota
+	ASCII DisplayMode = iota
+)
+
+type PrintOptions struct {
```

Comment:
I would prefer to not have this extra type `PrintOptions` wrapping the typedef `DisplayMode`. Just `DisplayMode` will do fine and also reject `int` arguments.